integrations/huggingface-transformers/tutorials/sparsifying_bert_using_recipes.md

```diff
@@ -0,0 +1,132 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Sparsifying Bert Models Using Recipes
+
+This tutorial presents essential extension from SparseML to HuggingFace Transformers training workflow to support model sparsification that includes knowledge distillation, parameter pruning, quantization and layer dropping. The examples used in this tutorial are specifically for Bert base uncased model trained and pruned on the Squad dataset, and further support and results will be available for other dataset in the near future.
+
+All the results listed in this tutorials are available publically through a [Weights and Biases project](https://wandb.ai/neuralmagic/sparse-bert-squad?workspace=user-neuralmagic).
+
+
```

Comment:
##Need Help?##
For Neural Magic Support, sign up or log in to get help with your questions in our **Tutorials channel:** [Discourse Forum](https://discuss.neuralmagic.com/) and/or [Slack](https://join.slack.com/t/discuss-neuralmagic/shared_invite/zt-q1a1cnvo-YBoICSIw3L1dmQpjBeDurQ).